cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/SubjectProviderOptions.java

```diff
@@ -0,0 +1,19 @@
+package org.opencds.cqf.fhir.cr.measure;
+
+/**
+ * All options that are pertinent to {@link org.opencds.cqf.fhir.cr.measure.common.SubjectProvider}
+ * implementations, such as {@link org.opencds.cqf.fhir.cr.measure.r4.R4RepositorySubjectProvider}.
+ */
+public class SubjectProviderOptions {
+
+    private boolean isPartOfEnabled;
```

Comment:
should have a javaDoc or comment explaining what functionality this option enables